src/ViewItem.js

```diff
@@ -93,6 +95,7 @@ class ViewItem extends React.Component {
 
     this.craftLayerUrl = craftLayerUrl.bind(this);
     this.cViewMetaData = props.stripes.connect(ViewMetaData);
+    this.getActionMenu = this.getActionMenu.bind(this);
```

Comment:
With modern JS class syntax we can get rid of binding methods. All that is needed is just to write methods in such way: `getActionMenu = () => {}`

Reply:
I love arrow functions but I just kinda made it a habit to avoid using them since I read this article (interesting read!):
https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1

The main reasons:
1. They are slower
2. They can hurt testability

But sure – I'll change it into an arrow function since the rest of the methods in this class are declared as arrow functions anyway.

Reply:
Yes, arrow functions are the tiniest bit slower at the moment, but I don't expect that to continue to be the case as browser vendors optimize for it. We don't really have to worry about the testability bit, since we're not writing white box component unit tests like described in that post.

Reply:
Got it. I changed all binded methods into arrow functions.